CryptoPrices/Sources/Home/Sources/Home/Localization/Strings.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  Strings.swift
+//  Home
+//
+//  Created by Minh Pham on 06/12/2022.
+//
+
+import Foundation
+
+public enum Strings {
+
+    enum Home {
+        @Localizable static var titleText = "home.title.text"
+        @Localizable static var totalCoinsTitle = "home.totalCoins.title"
+        @Localizable static var todayProfitTitle = "home.todayProfit.title"
+    }
+}
```

Comment:
@Thieurom I have tried to set up the [R.Swift](https://github.com/mac-cain13/R.swift) library for code generation but the R.generated.swift file fails to generate with the current SPM modular architecture as it was unable to detect and generate any resources in the SPM package. It would be great if you can try it for yourself or have a better suggestion regarding this problem, please also let me know 🙇‍♂️

Reply:
We can address this together with [the above issue](https://github.com/nimblehq/nimble-crypto-ios/pull/28#discussion_r1042005879) in the same task later.